src/cmd/serve.rs

```diff
@@ -759,6 +759,7 @@ pub fn serve(
 }
 
 fn is_ignored_file(ignored_content_globset: &Option<GlobSet>, path: &Path) -> bool {
+    // NOTE: globsets SHOULD be always Some unless config init fails, so we could unwrap
```

Comment:
I don't think we need that note, it implies knowledge about other parts of the code. The function takes an `Option` so we should still check. Otherwise pass the GlobSet directly without the option

Reply:
done, I removed the comment and left the match